src/content/docs/en/recipes/minification.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Minify HTML responses with middleware
+description: Use Astro middleware to minify HTML responses
+i18nReady: false
+type: recipe
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Astro supports built-in [HTML minification](/en/reference/configuration-reference/#compresshtml) which works for `.astro` components, but not for [islands](/en/concepts/islands/). This recipe shows how to use Astro middleware to minify complete HTML responses.
```

Comment:
I think this could be a *little* more connect the dots.

The lede is that Astro supports built-in minification... but then we're installing something else to minify? Can you make the connection stronger here between "not working for islands" and "a complete HTML response?"

I'm assuming that the idea is something like Astro will minify on a *per-component* basis, but only for `.astro` components. Since your islands are NOT getting minified by Astro's own compression, your HTML could get more mini! This external package can minify a complete page's HTML Response, and is how you can get HTML minification site-wide, at the page level, which would therefore include your islands.

I think the link missing is from "islands not minified" to "use middleware to minimize HTML Responses". It might not be *immediately* clear that this is the way to include islands. And, I think starting with the statement about what *sounds* like the thing they are going to do in this recipe (minify things!) being built in doesn't set the immediate expectation of installing another package.